tests/components/markers/markerIcon-test.js

```diff
@@ -273,7 +277,8 @@ describe('MarkerIcon component', function () {
     
     markerIcon.find('div.oo-marker-bubble.oo-marker-text').simulate('click');
     expect(seeked).toBe(true);
-    expect(seekTime).toBe(markerIcon.props().data.start);    
+    expect(seekTime).toBe(markerIcon.props().data.start);
+    expect(playheadUpdated).toBe(true);  
```

Comment:
There is a one more way to check if function was called. You could mock the function
```
updateSeekingPlayhead: jest.fn()
```
and after that use [toHaveBeenCalled](https://jestjs.io/docs/en/expect#tohavebeencalled) 
or [toHaveBeenCalledWith](https://jestjs.io/docs/en/expect#tohavebeencalledwith)
or .not.toHaveBeenCalled , etc

Reply:
Thanks, I'll use it next time =)

Reply:
@KseniiaSokolovaEpam let's leave it for now and refactor tests later